gen_errata.rb

```diff
@@ -81,11 +80,11 @@ def add_cve(cve)
 
   def add_package(name, version, architecture: nil, release: nil, component: nil)
     hsh = {
-      name: name,
-      version: version,
-      architecture: architecture,
-      release: release,
-      component: component
+      name:,
```

Comment:
why to use this format instead of name: name?

Reply:
rubocop :shrug: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashSyntax

We can force to not use shorthand syntax, though :innocent: